nowcasting_datamodel/read/read_gsp.py

```diff
@@ -57,6 +59,10 @@ def get_latest_gsp_yield(
         # filter on datetime
         query = query.where(GSPYieldSQL.datetime_utc >= datetime_utc)
 
+    # don't get any nans. (Note nan+1 > nan = False)
+    if no_nans_in_capacities:
+        query = query.where(GSPYieldSQL.capacity_mwp + 1 > GSPYieldSQL.capacity_mwp)
```

Comment:
How come you can't just query against NULLs? Are NaNs not saved as NULLs in the DB?

Reply:
thats the only way ive found it, but there could be another way

Reply:
I think they NaNs and Nulls are saved differently in postgres